src/fights.ts

```diff
@@ -821,6 +835,42 @@ const freeFightSources = [
     }
   ),
 
+  new FreeFight(
+    () =>
+      have($item`[glitch season reward name]`) &&
+      have($item`unwrapped knock-off retro superhero cape`) &&
+      !get("_glitchMonsterFights") &&
+      get("garbo_fightGlitch", false) &&
+     sum([...stunDurations], ([thing, duration]) => have(thing) ? undelay(duration) : 0) >= 5
+    () =>
+      withMacro(
+        Macro.trySkill($skill`Curse of Marinara`)
+          .trySkill($skill`Shadow Noodles`)
+          .trySkill($skill`Entangling Noodles`)
+          .trySkill($skill`Summon Love Gnats`)
+          .trySkill($skill`Frost Bite`)
+          .trySkill($skill`Soul Bubble`)
+          .tryItem($item`Rain-Doh blue balls`)
+          .skill($skill`Blow a Robo-Kiss`)
```

Comment:
Uh this is a Yellow Ray and a not free one at that.
I think you meant to use Heck General Tendrils which just requires configuring and wearing the retrocape.
https://kol.coldfront.net/thekolwiki/index.php/Unwrapped_knock-off_retro_superhero_cape

Reply:
Robo kiss is the correct one, no?

[](https://kol.coldfront.net/thekolwiki/index.php/Blow_a_Robo-Kiss)

Reply:
Unleash the Devil's Kiss is the yellow ray.

Reply:
Robo-kiss does 20% HP as damage but doesn't stun. The multi-round stun is Heck General/Hold Me.

Reply:
The plan is to stack up 5 rounds of consecutive stuns, and then use robo-kiss while the monster is stunned to kill it